filebeat/input/log/harvester.go

```diff
@@ -333,7 +333,8 @@ func (h *Harvester) Run() error {
 				logp.Info("File is inactive: %s. Closing because close_inactive of %v reached.", h.state.Source, h.config.CloseInactive)
 			case reader.ErrLineUnparsable:
 				logp.Info("Skipping unparsable line in file: %v", h.state.Source)
-				//line unparsable, go to next line
+				//line unparsable, update offset and go to next line
+				h.state.Offset += int64(message.Bytes)
```

Comment:
Should we also update read offset metrics as some lines below? https://github.com/elastic/beats/blob/015d379ad0c88ed8635f659d5c1fdc2c68059db1/filebeat/input/log/harvester.go#L360-L361

Reply:
@urso it would be good to have your input here

Reply:
Yes, these two should always be kept in sync.